components/identity-core/org.wso2.carbon.identity.core/src/main/java/org/wso2/carbon/identity/core/internal/DefaultServiceURLBuilder.java

```diff
@@ -203,6 +205,8 @@ private String fetchAbsoluteUrl() throws URLBuilderException {
                 absoluteUrl.append(":").append(port);
             }
 
+            appendContextToUri(absoluteUrl, proxyContextPath);
```

Comment:
A small suggestion.
I noticed that some parts of URL building logic is duplicated between fetchAbsoluteUrl and fetchRelativeUrl methods.

Can't we refactor the fetchAbsoluteUrl method to call the fetchRelativeUrl method internally?

Reply:
Yes. Will do that.